### PR TITLE
Change h3 to typography

### PR DESCRIPTION
**Pull request by Rabi94**
I changed all the h3 elements to Typography components with variant="h3" and set the font size to 2rem to maintain the original font size.

### CODE REVIEW

**Review by LarsKemmann — CHANGES_REQUESTED** [review]
The change to `<Typography>` is good, and we want to use the `h3` variant. But, the font size is too big and we don't want to set it everywhere. Please set it in the global Material UI theme settings, and match (roughly) the font size for `h6`.

**Comment by LarsKemmann** [comment]
https://v5.mui.com/material-ui/customization/theming/
https://v5.mui.com/material-ui/customization/typography/

File: src/caretogether-pwa/src/theme.tsx [hunk 1]
```diff
@@ -10,4 +11,14 @@ export const theme = createTheme({
       main: amber//'#ffc400'
     }*/,
   },
+  typography: {
+    h3: {
+      fontSize: '1.3rem',
+      '@media (min-width:600px)': {
+        [createTheme().breakpoints.up('sm')]: {
```
Comment (PabloDinella) [inline src/caretogether-pwa/src/theme.tsx:18]: I think you forgot to remove the line 17, as both 17 and 18 do the same thing

**Review by PabloDinella — CHANGES_REQUESTED** [review]

**Review by LarsKemmann — APPROVED** [review]
Looks great @Rabi94 and @PabloDinella - thank you!

**Review by PabloDinella — APPROVED** [review]

**Review by LarsKemmann — APPROVED** [review]

**Review by LarsKemmann — APPROVED** [review]

**Pull request merged**